detox/android/rninfo.gradle

```diff
@@ -42,4 +42,9 @@ ext.rnInfo = [
     isRN73OrHigher: rnMajorVer >= 73,
     isRN74OrHigher: rnMajorVer >= 74,
     isRN75OrHigher: rnMajorVer >= 75,
+    isRN76OrHigher: rnMajorVer >= 76,
```

Comment:
how about adding a method:
```
ext.rnInfo.gte(79)
```
(gte = Greater Than / Equal)

Naming not that important here